DOCS-10267 Asana Connector 1.0.0 (#1322)
mulesoft/docs-connectors: asana/1.0/modules/ROOT/pages/asana-connector-reference.adoc

= Asana Connector Module Connector Reference

> 

Author: Need to add the intro paragraph

Reviewer: Fixed.

== Configurations

[%header%autowidth.spread]
|===
| Name | Type | Description | Default Value | Required
|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
| Connection a| * <<Config_Oauth2, Oauth2 Connection Provider>>
* <<Config_Unsecured, Unsecured Connection Provider>>
 | The connection types to provide to this configuration. | | x
| Response Timeout a| Number |  The timeout for request to the remote service. |  1 |
| Response Timeout Unit a| Enumeration, one of:

** NANOSECONDS
** MICROSECONDS
** MILLISECONDS
** SECONDS
** MINUTES
** HOURS

> ** DAYS |  A time unit which qualifies the Response Timeout} |  MINUTES |

Author: ```suggestion
** DAYS |  Time unit that qualifies the Response Timeout |  MINUTES |
```

Author: Do a S&R for this one.

Reviewer: Fixed.
